docs/system-reqs.md

```diff
@@ -3,24 +3,24 @@
 The v1.0 or later version of 3PAR Volume plugin of Docker Engine Managed plugin system is supported and available now on Docker store (https://store.docker.com/plugins/hpe-3par-docker-volume-plugin/).
 Versions starting at v1.0 have been tested and are supported on the following Linux distributions:
 
-- Ubuntu 16.04
-- RHEL 7.3
-- CentOS 7.3
+* Ubuntu 16.04 and later
+* RHEL 7.3 and later
+* CentOS 7.3 and later
 
-Although the plugin software is supported on these linux distributions, you should consult the Hewlett Packard Enterprise Single Point of Connectivity Knowledge (SPOCK) for HPE Storage Products for specific details about which operating systems are supported by HPE 3PAR StoreServ and StoreVirtual Storage products (https://www.hpe.com/storage/spock).
+>**NOTE:** Although the plugin software is supported on the listed Linux distributions, you should consult the Hewlett Packard Enterprise Single Point of Connectivity Knowledge (SPOCK) for HPE Storage Products for specific details about which operating systems are supported by HPE 3PAR StoreServ and StoreVirtual Storage products (https://www.hpe.com/storage/spock).
+>
+>Although other Linux distributions have not been tested, the containerized version of the plugin should work as well.
 
-NOTE: Although other linux distributions have not been tested, the containerized version of the plugin should work as well.
+**Supported software versions:**
 
-Docker EE 17.03 or later is supported.
```

Comment:
Python 3.x is moved now, we support both 2.x and 3.x version, bTW by end of this year 2.x support will be stopped by Python..